src/remoteplayer.h

```diff
@@ -139,6 +141,16 @@ class RemotePlayer : public Player
 
 	void setPeerId(session_t peer_id) { m_peer_id = peer_id; }
 
+protected:
+	virtual bool checkPrivilege(const std::string &priv) const;
+	virtual void triggerJumpEvent(){/* noop in remote */};
```

Comment:
`virtual void triggerJumpEvent() { /* noop in remote */ };`

Reply:
Changed.